src/Confluent.Kafka/Consumer.cs

```diff
@@ -27,22 +27,15 @@
 
 namespace Confluent.Kafka
 {
-    /// <summary>
-    ///     Implements a high-level Apache Kafka consumer (with 
-    ///     key and value deserialization).
-    /// </summary>
-    public class Consumer<TKey, TValue> : IDisposable
+    /// <inheritdoc cref="IConsumer{TKey,TValue}"/>
```

Comment:
As far as I know, this project doesn't use sandcastle, and inheritdoc  is a sandcastle option.
Either wait for @mhowlett to see if sandcastle can be integrated, or use include and the indlude_docs.xml file. This would also allow to factorize some doc for Producer and non typed Consumer, so I would go for it